New PR to replace existing for Aggregations.io (#5893)
segmentio/segment-docs: src/connections/destinations/catalog/actions-aggregations-io/index.md

---
title: Aggregations.io (Actions) Destination
id: 659eb601f8f615dac18db564
beta: true
---

{% include content/plan-grid.md name="actions" %}

> [Aggregations.io](https://aggregations.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} enables you to utilize your existing analytics events & pipeline for real-time monitoring & alerting.

Reviewer: ```suggestion
[Aggregations.io](https://aggregations.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} enables you to use your existing analytics events and pipeline for real-time monitoring and alerting.
```

This destination is maintained by Aggregations.io. For any issues with the destination, [contact their Support team](mailto:help@aggregations.io).

## Getting started

1. From the Segment web app, click **Catalog**, then click **Destinations**.
2. Find the Destinations Actions item in the left navigation, and click it.
3. Click **Configure Aggregations.io (Actions)**.
4. Select an existing Source to connect to Aggregations.io (Actions).

> 5. In the destination settings, enter your Aggregations.io API Key & Ingest Id. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects` and the API Key requires `Write` permission.  For more information, see the [Aggregation.io Docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.

Reviewer: ```suggestion
5. In the destination settings, enter your Aggregations.io API Key and Ingest ID. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects`, and the API Key requires `Write` permission.  For more information, see the [Aggregation.io docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.
```

> 5. In the destination settings, enter your Aggregations.io API Key & Ingest Id. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects` and the API Key requires `Write` permission.  For more information, see the [Aggregation.io Docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.

Reviewer: ```suggestion
5. In the destination settings, enter your Aggregations.io API Key and Ingest ID. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects` and the API Key requires `Write` permission.  For more information, see the [Aggregation.io docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.
```
